<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoyalRoder Saver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>791</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>게시판 ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2409</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_id">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>설정 저장하기</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_t">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>rps</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>강사 ID</string>
      </property>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>241</width>
       <height>291</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>특정 게시판 저장</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_partsave">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>저장하기</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>시작</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>끝</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ptsave_st">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ptsave_ed">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stop">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>강제 종료</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>모든 게시판 저장</string>
      </attribute>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>410</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>저장하기</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>241</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>복사</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>붙여넣기</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>엔터</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>닫기</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="copy_sleep">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1.5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="paste_sleep">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="enter_sleep">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="close_sleep">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sleep">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>설정 저장하기</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>대기 시간 설정</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>421</width>
       <height>731</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>301</width>
        <height>411</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>장중교육</string>
       </attribute>
       <widget class="QListWidget" name="id_market">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>361</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>굴림</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>[비중법사] ID - victory</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2209</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2210</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2212</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>실습종목 : 2213</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>실습종목 현황 : 2240</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[주식신공] ID - singong</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2175</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2176</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2181</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>관심종목 : 2179</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[자그마치] ID - master</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2261</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2262</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2264</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>실습종목 : 2265</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>요약게시판 : 2413</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[프로장인] ID - pro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2395</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2396</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2398</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>실습종목 : 2399</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>저녁교육</string>
       </attribute>
       <widget class="QListWidget" name="id_evening">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>361</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>굴림</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>[오성] ID - dream590</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2026</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2027</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>관심공략주 : 2147</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[확률승부] ID - pppp1245</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2116</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2117</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>관심종목 : 2141</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[푸른별] ID - bluestar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2219</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2210</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2212</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>실습종목 : 2213</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>실습종목 현황 : 2240</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[수급쎄오] ID - ceo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2386</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2387</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2389</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>관심종목 : 2390</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>쎄오그램 : 2393</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>트레이닝스쿨</string>
       </attribute>
       <widget class="QListWidget" name="id_training">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>361</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>굴림</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>[서희파더] ID - toptrader</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2381</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2382</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2384</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>관심종목 : 2385</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>질문게시판 : 2416 - 코드 처리 안함</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[제시스페라] ID - spera</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2283</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2284</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2286</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>초보자 게시판 : 2414</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>질문 게시판 : 2415</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[포인트지지] ID - point</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2293</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2294</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2296</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>실습종목 : 2297</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[한백] ID - hanbaek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2401</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2403</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>실습종목 : 2404</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>[리버스패스] ID - rps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>공지사항 : 2409</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>회원게시판 : 2410</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강의후기 : 2411</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강사칼럼 : 2412</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>관심종목 : 2417</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>질문게시판 : 2418</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강의자료 : 2419</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>361</width>
        <height>271</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:11.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;[사용방법]&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;1. 게시판 ID, 강사 ID에 해당 값을 넣어주고&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;&amp;quot;설정 저장하기&amp;quot; 버튼을 누릅니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;2.&amp;quot;특정 게시판 저장&amp;quot; 탭을 클릭해 원하는 게시판&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;범위를 설정합니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;3. &amp;quot;저장하기&amp;quot; 버튼을 클릭하면 자동으로 저장됩니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;4. 만약 오류가 나면 &amp;quot;강제 종료&amp;quot; 버튼으로 종료할 수 있습니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;5. 대기 시간 설정으로 각 행동마다 대기시간을 조정할 수 있습니다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;6. &amp;quot;설정 저장하기&amp;quot; 버튼을 클릭해야지 적용됩니다.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>740</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>made by lastine_</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
